.buildkite/scripts/steps/publish_oas_docs.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+## Install the bump CLI for just this step
+npm install -g bump-cli@^2.8.2
```

Comment:
I think it doesn't hurt to have this added to the package.json of the tools, it wouldn't really increase the install time that much